pkg/env/prod.go

```diff
@@ -271,6 +271,10 @@ func (p *prod) OIDCEndpoint() string {
 	return fmt.Sprintf("https://%s/", os.Getenv("OIDC_AFD_ENDPOINT"))
 }
 
+func (p *prod) OIDCKeyBitSize() int {
```

Comment:
This might be considered "abuse" of the env.Interface because the value is hard-coded - at the same time it felt like the easiest plug-and-play solution to get a [steps.Action call configured properly](https://github.com/Azure/ARO-RP/blob/710eb2bdf0dc02772066b48a5111000367f48e0e/pkg/cluster/install.go#L301). Please LMK if there are alternative ways I'm not thinking of to inject this configuration :)

Reply:
Since it is needed to be tested and passed from https://github.com/Azure/ARO-RP/blob/d6f776e05f72c5558815a1bb57c7e5974dde27ec/pkg/cluster/deploybaseresources_test.go, I too think `env` can be the easiest way to configure the value. Better than inserting an if case for the localDev.